docs/dev-env-first-time-contributors.md

```diff
@@ -190,11 +190,16 @@ sudo apt-get install build-essential git ruby lxc redir
 If you do, make sure to **install default required packages** along with
 **git**, **curl**, **openssh**, and **rsync** binaries.)
 
-After installing, you must run **Git BASH as an administrator**.
-
 Also, you must have hardware virtualization enabled (VT-X or AMD-V) in your
 computer's BIOS.
 
+#### Running Git BASH as an administrator
+
+It is important that you **always run Git BASH with administrator privileges**
+when working on Zulip code, as not doing so will cause errors in the development
+environment (such as symlink creation). You might wish to configure your Git
+BASH shortcut to always run with these privileges enabled.
```

Comment:
thought about doing that, but it's actually not that simple (see [this answer](https://superuser.com/questions/1002262/run-applications-as-administrator-by-default-in-windows-10)). is it still worth linking to it?